spec/rainforest_auth_spec.rb

```diff
@@ -43,22 +43,22 @@
     end
 
     it "returns the expected signature" do
-      @auth.sign('test', {:option => 1}).should == '5957ba2707a51852d32309d16184e8adce9c4d8e'
+      @auth.sign('test', {:option => 1}).should == '65f2253344287b3c5634a1ce6163fb694b2280b1'
```

Comment:
Yes, it's now using the signed key instead of the key, so the signature is different.